control/control.openSUSE.xml

```diff
@@ -829,6 +827,11 @@ TODO: prepare disk here when it does not break space calculation (and remove bel
             <mode>update</mode>
             <stage>initial</stage>
             <modules config:type="list">
+                <!-- As soon as possible -->
```

Comment:
In SLES/SLED, at this point, the network configuration is done. That's no the case in openSUSE.

Reply:
This was commented but never added and looks like missing